cfme/tests/v2v/test_v2v_schedule_migrations.py

```diff
@@ -0,0 +1,71 @@
+"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
+import fauxfactory
+import pytest
+
+from cfme.fixtures.provider import rhel7_minimal
```

Comment:
No need to import this, the module is registered with pytest in `cfme/test_framework/pytest_plugin`

Reply:
I believed I had this comment by some one on my v2v PR, and I tried removing this, but didn't work.